<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>&lt;&gt;&lt; Recorder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="bottomButtonBox" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QPushButton" name="prevBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Go back to the previous slide</string>
         </property>
         <property name="text">
          <string>[◀] Previous</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="blankBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Blank the presentation screen.</string>
         </property>
         <property name="text">
          <string>[B] Blank</string>
         </property>
         <property name="shortcut">
          <string>B</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="nextBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Advance to the next slide</string>
         </property>
         <property name="text">
          <string>[▶] Next</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="monitorBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>[M] Monitor</string>
         </property>
         <property name="shortcut">
          <string>M</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="recordBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Starts or stops the recording of audio. This is not an instant operation, there might be latencies of up to a second involved.</string>
         </property>
         <property name="text">
          <string>[S] Record</string>
         </property>
         <property name="shortcut">
          <string>S</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="freezeBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Freeze the picture currently shown on the presentation screen</string>
         </property>
         <property name="text">
          <string>[F] Freeze</string>
         </property>
         <property name="shortcut">
          <string>F</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
